src/components/Homepage.js

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { BACKEND_URL } from "../constants";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import "../App.css";
+
+export default function Homepage() {
+  const [sightings, setSightings] = useState([]);
+
+  useEffect(() => {
+    const takeSightings = async () => {
```

Comment:
```suggestion
    const getSightings = async () => {
```

get them, don't take them :D